app/src/renderer/components/staking/ModalUnstake.vue

```diff
@@ -0,0 +1,108 @@
+<template lang="pug">
+  .modal-stake#modal-stake
+    //- Header
+    .stake-header
+      img.icon(class='stake-atom' src="~assets/images/cosmos-logo.png")
+      span.tm-modal-title Unstake
+      .tm-modal-icon.tm-modal-close(@click="close()")
+        i.material-icons close
+
+    //- To
+    tm-form-group.stake-form-group(
+      field-id='to' field-label='To')
+      tm-field#to(
+        readonly v-model="to")
+
+    //- Amount
+    tm-form-group.stake-form-group(
+      field-id='amount'
+      field-label='Amount'
+    )
+      tm-field#amount(
+        :max="maximum"
+        :min="0"
+        type="number"
+        v-focus
+        v-model="amount")
+
+    //- Footer
+    .stake-footer
```

Comment:
We don't have a close button on the stake modal. Should we remove this one as well ? (cc: @jbibla)

Reply:
yep